test/EnsembleKalmanProcesses/runtests.jl

```diff
@@ -77,6 +82,7 @@ using CalibrateEmulateSample.ParameterDistributionStorage
     # Plot evolution of the EKI particles
     eki_final_result = vec(mean(ekiobj.u[end], dims=1))
     
+    TEST_PLOT_OUTPUT = false
```

Comment:
> When running test/EnsembleKalmanProcesses/runtests.jl, it seems like TEST_PLOT_OUTPUT is not defined. Can you verify this behavior @odunbar ?

In theory we should run the tests as a block i.e from `test/runtests.jl` and so this is where TEST_PLOT_OUTPUT is defined (via a get method with 0 default if the CES_TEST_PLOT_OUTPUT variable is not defined). If you directly run the EKP block from `test/EnsembleKalmanProcesses/runtests.jl` you would need to define it. 

So i would remove the line you've added here. Otherwise these run for me! Thanks!

Reply:
Edit: I've removed this in latest commit.

Reply:
Great, thanks!